packaging/greenboot/red.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
This script duplicates code except the HEALTH setting. This setting can be deduced from the `red.d` or `green.d` directory name of the current script. Let's remove the duplication please?

Reply:
An earlier version of the code was deriving status from the path name and I asked to have that changed to be more explicit and less "tricky". We can share a lot of the body of the scripts, but I'd like us to have 2 scripts in git to make it easier to clearly manage any differences. For my part, we can improve on the code reuse in the next PR, unless @pmtk wants to go ahead and include it in this PR.

Reply:
It's the only way for a script to "know" at what state greenboon ran it, so I think this is pretty standard.
Anyway, at least the script names should be changed to be less generic.

Reply:
Sure, renaming the scripts would make sense. It would be nice if we could organize the git repo similarly to how the scripts will be installed to avoid confusion when making changes to the scripts.

Reply:
Renamed scripts, but kept them separate for now